component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1639,4 +1638,26 @@ private long getUnlockTimeInMilliSeconds(AuthenticatedUser authenticatedUser) th
                     username, e);
         }
     }
+
+    /**
+     * Returns AuthenticatedUser object from context.
+     *
+     * @param context AuthenticationContext.
+     * @return AuthenticatedUser
```

Comment:
```suggestion
     * @return AuthenticatedUser.
```

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/154/commits/a67fa057a7d8e1efb3b52117cad0f7fb71f7205b